package.json

```diff
@@ -22,7 +22,7 @@
 		"eslint-config-vuepress": "^2.1.0",
 		"eslint-plugin-prettier": "^3.1.4",
 		"prettier": "^2.0.5",
-		"vuepress": "^1.5.2",
+		"vuepress": "^1.8.2",
```

Comment:
I believe we may prefer to keep dependencies update in a separate pull request, maybe to the `master` branch? Unless there is a need of this update for this pull request. This would reduce a bit the amount of things that need to be tested.

Reply:
I tried to upgrade the version to fix the sidebar, which did not help. I mostly did so because vuepress github required latest version before filing an issue.